ensime_shared/ensime.py

```diff
@@ -944,9 +812,7 @@ def teardown(self):
 
     def current_client(self, quiet, bootstrap_server, create_client):
         """Return the current client for a given project."""
-        # Use current_file command because we cannot access self.vim
-        current_file_cmd = commands["current_file"]
-        current_file = self.vim.eval(current_file_cmd)
+        current_file = self.vim.current.buffer.name
```

Comment:
This should be delegated on a method in `editor` considering we ain't gonna reference `vim` directly from the client anymore.